packages/app/components/Settings/SettingsSharedComponents.tsx

```diff
@@ -2,13 +2,18 @@ import { useWindowWidth } from "@react-hook/window-size";
 import React, { ReactElement } from "react";
 import SelfAvatar from "../common/SelfAvatar";
 
-export function SettingsPanelAvatar(): ReactElement {
-  const avatarSize = useWindowWidth() / 10;
+export function SettingsPanelAvatar(props): ReactElement {
+  const { avatarSize } = props;
```

Comment:
imo destructuring the props at the function signature is a bit cleaner (like `function SettingsPanelAvatar({avatarSize})`)